app/components/polls/questions/answers_component.html.erb

```diff
@@ -1,16 +1,22 @@
 <div class="poll-question-answers">
   <% if can?(:answer, question) && !question.poll.voted_in_booth?(current_user) %>
     <% question_answers.each do |question_answer| %>
-      <% if already_answered?(question_answer) && !voted_before_sign_in? %>
-        <span class="button answered"
-              title="<%= t("poll_questions.show.voted", answer: question_answer.title) %>">
+      <% if already_answered?(question_answer) %>
+        <%= button_to question_answer_path(question, user_answer(question_answer)),
+                      method: :delete,
+                      remote: true,
+                      title: t("poll_questions.show.voted", answer: question_answer.title),
+                      class: "button answered expand",
```

Comment:
Does the `expand` class do anything? :thinking: Since it isn't present in the button to remove the question, I guess it's a typo.

```suggestion
                      class: "button answered",
```

Reply:
No. Removed.